### ISSUE TITLE
Too many branches

### ISSUE THREAD

**Issue by Zimmi48** [ocaml/dune#1654]
This project has a lot of branches, and this means that it is hard to follow the development. As can be seen in https://github.com/ocaml/dune/branches/all, most of these branches are associated to merged PRs. It would be nice to at least delete these branches. Probably some old and unused branches like `dev` too. I would personally recommend going further by having contributors open PRs from branches on their forks (including core contributors) but this is your call. Making the number of branches smaller would already be a good improvement.

**Comment by rgrinberg** [comment]
I don't have any objections to deleting old branches. I did not realize that people followed all the different branches.

**Comment by Zimmi48** [comment]
>I did not realize that people followed all the different branches.

That's not what I meant. I mean that want to look at your various branches to try to infer your model of development. For instance, you have a `dev` branch that has not been used for a long time but can be confusing because people used to the git-flow model might wrongly think that this is the branch that PR should target.

**Comment by emillon** [comment]
PRs should target master. I'll cleanup the merged branches so that it's less messy!

**Comment by emillon** [comment]
Done - I also clarified that bit in the contributing guide. Thanks!

**Comment by emillon** [comment]
(And I'll also try to make my PRs from my fork :))
